.travis.yml

```diff
@@ -36,6 +36,7 @@ jobs:
         - codecov
       env:
         - PYTHONPATH="${TRAVIS_BUILD_DIR}/app/"
+        - DJANGO_SETTINGS_MODULE="app.settings"
```

Comment:
Is this needed?

Reply:
Yes. Seems so. The environment variable declaration in setup.cfg under `tool:pytest` seems to not been working most probably in the newer versions. They need the environment variables to be set before the pytest can be run. Without this travis fails.